editor/scene_tree_dock.cpp

```diff
@@ -713,6 +713,14 @@ void SceneTreeDock::_tool_selected(int p_tool, bool p_confirm_override) {
 				}
 			}
 		} break;
+		case TOOL_GOTO_DOCUMENTATION: {
+
+			List<Node*> selection = editor_selection->get_selected_node_list();
```

Comment:
Small style issue for `clang-format` here: https://travis-ci.org/godotengine/godot/jobs/308827286

(not that clang-format's style is inherently better in this case, but with automated tools we can't easily make exceptions :))